stats/view.go

```diff
@@ -41,41 +42,23 @@ type View struct {
 	// start is time when view collection was started originally.
 	start time.Time
 
-	// ss are the channels through which the collected views data for this view
-	// are sent to the consumers of this view.
-	ss map[chan *ViewData]subscription
+	collectionMu sync.Mutex // guards forced and subscribed
+	forced       bool       // indicates whether the view should collect data if no one is subscribed
```

Comment:
Can we use atomic variables instead of a mutex here?

Reply:
Done